docs/reference/objects/product/extra.md

```diff
@@ -53,6 +53,10 @@ Returns the initial stock for the extra, if the extra has unlimited inventory th
 
 Returns a list of [modifier groups](({% link docs/reference/objects/product/modifier_group.md %})) associated with this extra.
 
+# extra.pre_checkout_modifier_groups
```

Comment:
Nice! I'd say maybe update the modifier_group to highlight that it include post and pre checkout modifiers, and reference this method there? 

We generally add methods in alphabetical order.

Reply:
Thanks @geoffreymm! good shouts will implement

Reply:
Addressed in https://github.com/easolhq/easolhq.github.io/pull/50/commits/18dab2ac28f352fd3cebea04dca11738f17d45a0 - let me know what you think

Reply:
Addressed in https://github.com/easolhq/easolhq.github.io/pull/50/commits/f66785447a5316a32e12d172ae1fc757cadcf92f - wdyt?